test/ut_expectations/test_expec_not_to_be_null.pks

```diff
@@ -0,0 +1,65 @@
+create or replace package test_expec_not_to_be_null
+is
+    --%suite(expectations - not_to_ve_null)
+    --%suitepath(utplsql.core.expectations.no_to_be_null)
+
+    --%test(test blob not null)
```

Comment:
No need to put "test" in the test description. We all know it's a test.
Best way to name tests is by describing behavior that they test. This way you have feature documentation straight from your tests.
Example:
`--%test(Gives success for a not null BLOB)